parsons/utilities/zip_archive.py

```diff
@@ -0,0 +1,46 @@
+import zipfile
+
+
+def create_archive(archive_path, file_path, file_name=None, if_exists='replace'):
+    """
+    Create and fill an archive.
+
+    `Args:`
+        archive_path: str
+            The file name of zip archive
+        file_path: str
+            The path of the file
+        file_name: str
+            The name of the file in the archive
+        if_exists: str
+            If archive already exists, one of 'replace' or 'append'
+    `Returns:`
+        Zip archive path
+    """
+
+    if if_exists == 'append':
+        write_type = 'a'
+    else:
+        write_type = 'w'
+
+    if not file_name:
```

Comment:
Of course I defer to your greater experience, but I wanted to raise that I've had difficulty using `if not` to rule out Nonetypes. I've often had to switch to `if variable is not None:`